app/src/main/java/org/p2p/wallet/restore/ui/seedphrase/SeedPhraseFragment.kt

```diff
@@ -57,6 +61,23 @@ class SeedPhraseFragment :
 
             seedPhraseView.focusAndShowKeyboard()
         }
+
+        val deeplinkUri = deeplinksManager.pendingDeeplinkUri ?: return
+        if (DeeplinkUtils.hasFastOnboardingDeeplink(deeplinkUri)) {
```

Comment:
i suggest creating a wrapper class for DeeplinkUri and check what is it as methods

 val deeplinkUri: DeeplinkUri = deeplinksManager.pendingDeeplinkUri ?: return
if (deeplinkUri.isFastOnboarding){
...
}